config/docs.config.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
   editorConfig: {
     theme: "night",
   },
+  simpleEditor: false,
```

Comment:
New option coming from new version - needs to be set otherwise the build will fail. https://vue-styleguidist.github.io/Configuration.html#editorconfig